modules/tinymce/src/plugins/visualchars/test/ts/atomic/HtmlTest.ts

```diff
@@ -1,17 +1,21 @@
-import { Assert, UnitTest } from '@ephox/bedrock-client';
+import { describe, it } from '@ephox/bedrock-client';
 import { Unicode } from '@ephox/katamari';
+import { assert } from 'chai';
+
 import * as Html from 'tinymce/plugins/visualchars/core/Html';
 
-UnitTest.test('atomic.tinymce.plugins.visualchars.HtmlTest', () => {
-  Assert.eq(
-    'should return correct span',
-    '<span data-mce-bogus="1" class="mce-nbsp">' + Unicode.nbsp + '</span>',
-    Html.wrapCharWithSpan(Unicode.nbsp)
-  );
+describe('atomic.tinymce.plugins.visualchars.HtmlTest', () => {
+  it('should return correct span with nbsp', () => {
```

Comment:
Likewise, I would have just expected the one test here, as this is testing the module not `wrapCharWithSpan`. I guess the alternative option would be to wrap it inside a `context` call to highlight that's what it's testing.

Reply:
Also worth noting most current unit tests, test the function generally not one specific aspect of it (eg `DomUtilsTest` is a good example).

Reply:
This just needs to be wrapped in a context block now given our discussion at standup.

- [ ] Wrap in context